docs/BUILD.md

```diff
@@ -39,27 +38,10 @@ For example, define the link to the release notes pointing to a released version
 
 > Additionally define important changes, breaking changes or important new features into the description.
 
-After the release build status is **success** follow the next steps below. Check [here](https://github.com/commercetools/commercetools-sync-java/actions?query=workflow%3ADEPLOY)
- for github action build status.
-
-## Step 2: Publish
-
-- Go to [https://bintray.com/commercetools/maven/commercetools-sync-java](https://bintray.com/commercetools/maven/commercetools-sync-java)
-- Click the _Publish_ button as shown below:
-
-<img width="600" alt="Screenshot-1" src="https://user-images.githubusercontent.com/3469524/86367568-671ad600-bc7c-11ea-84e9-786f9f850d96.png">
-
-## Step 3: Sync
-
-- Click to the Maven Central tab.
-- Enter the commercetools Sonatype Maven Central API Key and password (into User token key and User token password fields)
-- Click the _Sync_ button and check the `Sync Status`. 
-
-<img width="600" alt="Screenshot-2" src="https://user-images.githubusercontent.com/3469524/86331559-906e3e80-bc49-11ea-9390-e813bc12c163.png">
+The creation of a github release triggers a [github action](https://github.com/commercetools/commercetools-sync-java/actions?query=workflow%3ACD), which will deploy the library to [Maven Central](https://mvnrepository.com/artifact/com.commercetools/commercetools-sync-java).
 
 ## Final Step
-
-Ensure the new version is publicly available at [JCenter](https://bintray.com/commercetools/maven/commercetools-sync-java) and [Maven Central](https://mvnrepository.com/artifact/com.commercetools/commercetools-sync-java). 
+After the release build status is **success** ensure that the new version is publicly available at [Maven Central](https://mvnrepository.com/artifact/com.commercetools/commercetools-sync-java). 
 
 ### Checklist 
```

Comment:
I guess some parts of the checklist is obsolete now?

Reply:
7f4a3eb67

Reply:
I see duplicated text starts with: `The new version is available in the maven`

Reply:
cc8489db9